src/collectors/filestat/filestat.py

```diff
@@ -233,28 +309,20 @@ def process_lsof(self, users, types):
         Get the list of users and file types to collect for and collect the
         data from lsof
         """
-        d = {}
-        for u in users:
-            d[u] = {}
-            tmp = os.popen("lsof -wbu %s | awk '{ print $5 }'" % (
-                u)).read().split()
-            for t in types:
-                d[u][t] = tmp.count(t)
-        return d
+        return parse_lsof_output(self.get_output('tL'), users, types)
```

Comment:
Switched to `lsof -F tL`, because I found that the type is not always in `awk '{ print $5 }'`.